src/service/passport/local.js

```diff
@@ -42,13 +42,18 @@ const configure = async (passport) => {
 };
 
 /**
- * Create the default admin user if it doesn't exist
+ * Create the default admin and regular test users.
  */
 const createDefaultAdmin = async () => {
-  const admin = await db.findUser("admin");
-  if (!admin) {
-    await db.createUser("admin", "admin", "admin@place.com", "none", true);
-  }
+  const createIfNotExists = async (username, password, email, type, isAdmin) => {
+    const user = await db.findUser(username);
+    if (!user) {
+      await db.createUser(username, password, email, type, isAdmin);
+    }
+  };
+
+  await createIfNotExists('admin', 'admin', 'admin@place.com', 'none', true);
```

Comment:
Just a heads up - there are plans to improve/refactor the way default users are created: #1022 

I believe the reason why user registering wasn't implemented was that the first adopting organizations only needed ActiveDirectory for their user management. But for others, having default accounts could be a security concern (supposing a malicious actor got access to the GitProxy UI).

Happy to discuss other ways to improve this!